dev_guide/templates.adoc

```diff
@@ -51,67 +56,154 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
-== Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+== Creating a Configuration From a Template
+You can create a configuration from a template using the CLI or, if a template
+has been uploaded to your project or global template library, using the
+Management Console.
 
-There is also the ability to add labels in the template from the command line.
+Create a template JSON file, like the above example, then upload it with the
+CLI using the following process:
+
+=== Uploading a Template
+You can upload a template to your current project's template library by passing
+a JSON file with the following command:
 
 ****
-`$ osc process -f _<filename>_ -l name=otherLabel`
+`osc create -f _<filename>_`
 ****
 
-== Parameters
-The list of parameters that you can override are listed in the `*parameters*` section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
+You can upload a template to a different project using the `-n` option with the
+name of the project:
 
 ****
-`$ osc process --parameters -f _<filename>_`
+`osc create -f _<filename>_ -n _<project>_`
 ****
 
-The following shows the output when listing the parameters for one of the *_openshift/origin/examples_* templates.
-
-.Example Template Output
-====
+The template is now available to be selected for a configuration using the
+Management Console or the CLI.
 
-----
-$ osc process --parameters -f examples/sample-app/application-template-dockerbuild.json
-NAME                  DESCRIPTION              GENERATOR           VALUE
-ADMIN_USERNAME        administrator username   expression          admin[A-Z0-9]{3}
-ADMIN_PASSWORD        administrator password   expression          [a-zA-Z0-9]{8}
-MYSQL_ROOT_PASSWORD   database password        expression          [a-zA-Z0-9]{8}
-MYSQL_DATABASE        database name                                root
-----
+Alternatively, you can upload a template then create from the same template by
```

Comment:
that command isn't uploading the template, it's just processing it from the filesystem, so the template doesn't end up in the template library.